specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -769,7 +769,7 @@ interface for interactions with the user to gather authentication data.
 In addition, an application client may provide a class that implements
 the _javax.security.auth.callback.CallbackHandler_ interface and specify
 the class name in its deployment descriptor (see
-link:#a3399[See Java EE Application
+link:#a3399[See Jakarta EE Application
```

Comment:
Do these cross-chapter links still work after splitting the single document into multiple documents?

Reply:
Not sure, to be honest, @bshannon.  Haven't worried about it yet...  I'm sure we can make them work somehow.  Or, we can re-combine the sections if we have to...  My first goal is to just to get a new "Jakarta" baseline...

Reply:
@bshannon Turns out that it's no worse than it was ...  :-(  I just went back to the original single adoc version of the spec and the none of the hotlinks in the pdf work.  Same error with the old format as with the new format...  So, we need to figure out the proper reference syntax and do some more updating...  This is so much fun...

Reply:
Yes, there's a separate problem with the links in the pdf.  If someone could figure out
how to make that work that would be great.  But the links *did* work in the html.

Reply:
Just starting to look into this link issue...  Some of the links seem to have direct links to a corresponding .html page...  That's why many of them work in the generated html...  I'll work on a couple of updates, create a new PR, and get feedback before trying to update all of the references.